apps/model-ad/app/src/app/app.component.html

```diff
@@ -17,3 +17,4 @@
     [dataVersion]="dataVersion"
   ></explorers-footer>
 </div>
+<p-toast class="global-toast" position="bottom-center"></p-toast>
```

Comment:
In Agora, all toasts used this global toast. I've implemented a new toast specific to this button so that the toast appears near the button that was clicked. This global toast will be used for other events like errors and shows up at the bottom of the screen.